### PR TITLE
updating filestack to work when gatsby builds

### PR DESCRIPTION
**Pull request by gillkyle**
Based on the errors from #40, updated the filestack component to use [`@loadable/component`](https://github.com/smooth-code/loadable-components) to ignore the filestack import during `gatsby build`.

The only new changes made with this since it was first added are on [this line](https://github.com/gatsby-inc/gatsby-interface/compare/filestack-with-ssr?expand=1#diff-0b5d89c1ee435a092f509cd5e29bc7b3R8), after installing loadable.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@loadable/component": "^5.10.2",
```
Comment (gillkyle) [inline package.json]: Should this not be a regular dependency?

Comment (rizchelwood) [reply package.json]: It is a regular dependency?

**Comment by gillkyle** [comment]
Just changed the base branch to `dev` and fixed conflicts, thanks @rizchelwood!

**Comment by gillkyle** [comment]
According to @wardpeet, this is fine to be included as a regular dependency 👍

**Comment by rizchelwood** [comment]
Thanks @gillkyle! This looks good, will merge 👍

**Pull request merged**